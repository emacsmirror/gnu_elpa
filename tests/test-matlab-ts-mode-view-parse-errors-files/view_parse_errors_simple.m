% -*- matlab-ts -*-

a = = 2000;

b = *= 2;

