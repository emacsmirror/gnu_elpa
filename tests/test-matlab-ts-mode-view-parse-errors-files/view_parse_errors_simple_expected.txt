test-matlab-ts-mode-view-parse-errors-action-fun result:
---
Tree-sitter parse errors.
view_parse_errors_simple.m:3:3: error: parse error from line 3:3 to line 3:4 (point 24 to 25)
    3 | a = = 2000;
      |   ^
view_parse_errors_simple.m:5:3: error: parse error from line 5:3 to line 5:6 (point 37 to 40)
    5 | b = *= 2;
      |   ^
